src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Application {
+    private static final int FIGURE_COUNT = 6;
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[FIGURE_COUNT];
+        for (int i = 0; i < FIGURE_COUNT; i++) {
+            if (i < FIGURE_COUNT / 2) {
+                figures[i] = figureSupplier.getRandomFigure();
+            } else {
+                figures[i] = figureSupplier.getDefaultFigure();
+            }
+        }
+        for (Figure figure : figures) {
+            System.out.println(figure.draw());
```

Comment:
Let's print info inside of 'draw' method 
```suggestion
            figure.draw();
```